main.tf

```diff
@@ -116,22 +125,37 @@ resource "aws_instance" "default" {
   tags = "${module.label.tags}"
 }
 
+##
+## Create keypair if one isn't provided
+##
+
+module "ssh_key_pair" {
+  source                = "git::https://github.com/cloudposse/terraform-aws-key-pair.git?ref=master"
```

Comment:
pin to a release

Reply:
@Jamie-BitFlight 
please address the comments in https://github.com/cloudposse/terraform-aws-key-pair/pull/13 so we understand the purpose of using this module here

Reply:
No prob. Will pin if a release is done in cloudposse/terraform-aws-key-pair#13